cmd/draft/create.go

```diff
@@ -1,4 +1,4 @@
-package main
+	package main
```

Comment:
mind moving this back? `package` doesn't need to be indented. `go fmt ./...` oughta do the trick. :)

Reply:
@bacongobbler d'oh; probably leftover from a merge - should be fixed now.